python/MooseDocs/extensions/appsyntax.py

```diff
@@ -689,87 +691,108 @@ def _createHTMLHelper(self, parent, token, page, func, collapse):
-                latex.Command(args[2], 'texttt', string=param['cpp_type'])
-                default = _format_default(param) or ''
-                if default:
-                    args.append(latex.Bracket(string=default))
-
-                latex.Environment(parent, 'InputParameter',
-                                  args=args,
-                                  string=param['description'])
+                self.renderer.render(parent, param, page)
 
 class RenderParameterToken(components.RenderComponent):
 
     def createHTML(self, parent, token, page):
-        return html.Tag(parent, 'span', class_='moose-parameter-name')
+        param = token['parameter']
+        name = param['name']
+
+        if param['deprecated']:
+            return
+
+        li = html.Tag(parent, 'li')
+        default = _format_default(param)
+
+        if default:
+            html.Tag(li, 'strong', string=name)
+            html.Tag(li, 'span', string=' ({}): '.format(default))
+        else:
+            html.Tag(li, 'strong', string='{}: '.format(name))
+
+        desc = param['description']
+        if desc:
+            html.Tag(li, 'span', string=desc)
 
     def createMaterialize(self, parent, token, page):
-        span = self.createHTML(parent, token, page)
         param = token['parameter']
-        span.addClass('tooltipped')
-        span['data-tooltip'] = param['description']
-        span['data-position'] = 'bottom'
-        span['data-delay'] = 5
-        return span
+        name = param['name']
+
+        if param['deprecated']:
+            return
+
+        default = _format_default(param)
+        desc = param['description']
+
+        if token['include_heading']:
+            li = html.Tag(parent, 'li')
+            header = html.Tag(li, 'div', class_='collapsible-header')
+            body = html.Tag(li, 'div', class_='collapsible-body')
+
+            html.Tag(header, 'span', class_='moose-parameter-name', string=name)
+            if default:
+                html.Tag(header, 'span', class_='moose-parameter-header-default', string=default)
+
+            if desc:
+                html.Tag(header, 'span', class_='moose-parameter-header-description', string=str(desc))
+
+        else:
+            body = parent
+
+        if default:
+            p = html.Tag(body, 'p', class_='moose-parameter-description-default')
+            html.Tag(p, 'span', string='Default:')
+            html.String(p, content=default)
+
+        cpp_type = param['cpp_type']
+        p = html.Tag(body, 'p', class_='moose-parameter-description-cpptype')
+        html.Tag(p, 'span', string='C++ Type:')
+        html.String(p, content=cpp_type, escape=True)
+
+        if 'options' in param:
+            p = html.Tag(body, 'p', class_='moose-parameter-description-options')
+            html.Tag(p, 'span', string='Options:')
+            html.String(p, content=param['options'])
```

Comment:
I vote for a comma between the options.
```suggestion
            html.String(p, content=", ".join(param['options'].split()))
```

Reply:
Thanks Andrew. I went ahead and added the commas.